lib/data/multipleBackendGateway.js

```diff
@@ -216,6 +218,90 @@ const multipleBackendGateway = {
             return callback(null, multBackendResp);
         });
     },
+
+    createMPU: (key, bucket, metaHeaders, websiteRedirectHeader,
+    location, log, cb) => {
+        const client = clients[location];
+        const awsKey = _createAwsKey(bucket, key, client.bucketMatch);
+        if (client.clientType === 'aws_s3') {
+            const params = { Bucket: client.awsBucketName, Key: awsKey,
+                WebsiteRedirectLocation: websiteRedirectHeader,
+                Metadata: metaHeaders };
+            return client.createMultipartUpload(params, (err, data) => {
```

Comment:
let's not call this 'data'.  It makes it appear like we are actually getting the data back.